modules/mod_facebook/controllers/controller_facebook_redirect.erl

```diff
@@ -133,9 +133,28 @@ decode_access_token("application/json"++_, Payload) ->
 decode_access_token(_ContentType, Payload) ->
     mochiweb_util:parse_qs(Payload).
 
+% Given user id
+fetch_user_id(AccessToken) ->
+    FacebookUrl = "https://graph.facebook.com/v2.9/me?access_token=" ++ z_url:url_encode(AccessToken),
+    case httpc:request(FacebookUrl) of
+        {ok, {{_, 200, _}, _Headers, Payload}} ->
+            {struct, Props} = mochijson:binary_decode(Payload),
+            case proplists:get_value(<<"id">>, Props) of
+                undefined ->
+                    lager:error("[facebook] missing param id ~p", [Props]),
+                    {error, {http_error, FacebookUrl}};
+                _id ->
+                    {ok, jiffy:decode(_id)}
```

Comment:
This looks like a double decode: on line 141 we already have `mochijson:binary_decode/1`. Is this intentional?